src/cmd-import

```diff
@@ -228,6 +239,28 @@ def finalize_build(builds, build_meta, tmp_oci_archive, tmp_oci_manifest, tmp_lo
     print(f'Imported OCI image as build {buildid}')
 
 
+def calculate_and_log_diffs(builds, buildid, arch, starting_buildid):
+    # Get the RPM/Advisory diff info
+    diff_cmd_json = ['/usr/lib/coreos-assembler/cmd-diff', '--rpms-json',
+                     '--from', starting_buildid, '--to', buildid]
+    diff_json_str = subprocess.check_output(diff_cmd_json)
+    diff_data = json.loads(diff_json_str)
+
+    # Update the meta.json file with the RPM/Advisory diff info. Yes we
+    # just created this file, but let's use the library now for access.
+    meta = GenericBuildMeta(workdir=os.getcwd(), build=buildid, basearch=arch)
+    meta.update({
+        'pkgdiff': diff_data.get('pkgdiff'),
+        'advisories-diff': diff_data.get('advisories')
+    })
+    meta.write()
```

Comment:
Hmm right, I was thinking that we could directly write the updated `meta.json` on first shot, but indeed `cosa diff` doesn't know how to work with staged builds. And meh... doesn't seem worth it for this.